include/vctr/Containers/VctrBase.h

```diff
@@ -247,6 +247,28 @@ class VctrBase : private Config,
         std::copy_n (otherData, otherSize, begin());
     }
 
+    /** Fills the container with the given value. */
+    constexpr void fill (const value_type& value) {  std::fill (storage.begin(), storage.end(), value); }
+
+    /** Resizes this container to newSize and fills it with the given value.
+
+        @see resize, fill.
+     */
+    constexpr void init (size_t newSize, const value_type& value) { storage.resize (newSize); fill (value);  }
```

Comment:
In contrast to fill, init should only be available to `vctr::Vector` since it relies on the container having a resize function which is only the case for `vctr::Vector`. So please move the two init functions back to that class.